NuGet/Realm.Database/Realm.Database.nuspec

```diff
@@ -31,14 +31,18 @@
     <file src="../../Platform.PCL/Realm.PCL/bin/$Configuration$/Realm.*" target="lib/portable-net45+sl5+wp8+wpa81+win8+monoandroid+Xamarin.iOS10+monotouch+Xamarin.Mac" />
 <!-- IOS file -->
     <file src="../../Platform.XamarinIOS/Realm.XamarinIOS/bin/iPhoneSimulator/$Configuration$/Realm.*" target="lib/Xamarin.iOS10" />
-    <file src="../../wrappers/build/$Configuration$-ios-universal/librealm-wrappers.a" target="lib/Xamarin.iOS10" />
+    <file src="../../wrappers/build/$Configuration$-ios-universal/librealm-wrappers.a" target="native/ios/universal" />
 <!-- Targets file adds references to native libs and copies RealmWeaver -->
     <file src="Realm.Database.targets" target="build" />
 <!-- Android files -->
     <file src="../../Platform.XamarinAndroid/Realm.XamarinAndroid/bin/$Configuration$/Realm.*" target="lib/MonoAndroid44" />
-    <file src="../../wrappers/build/$Configuration$-android/armeabi-v7a/librealm-wrappers.so" target="lib/MonoAndroid44/armeabi-v7a" />
-    <file src="../../wrappers/build/$Configuration$-android/arm64-v8a/librealm-wrappers.so" target="lib/MonoAndroid44/arm64-v8a" />
-    <file src="../../wrappers/build/$Configuration$-android/x86/librealm-wrappers.so" target="lib/MonoAndroid44/x86" />
-    <file src="../../wrappers/build/$Configuration$-android/x86_64/librealm-wrappers.so" target="lib/MonoAndroid44/x86_64" />
+    <file src="../../wrappers/build/$Configuration$-android/armeabi-v7a/librealm-wrappers.so" target="native/android/armeabi-v7a" />
+    <file src="../../wrappers/build/$Configuration$-android/arm64-v8a/librealm-wrappers.so" target="native/android/arm64-v8a" />
+    <file src="../../wrappers/build/$Configuration$-android/x86/librealm-wrappers.so" target="native/android/x86" />
+    <file src="../../wrappers/build/$Configuration$-android/x86_64/librealm-wrappers.so" target="native/android/x86_64" />
+    <!-- Win32 files -->
+    <file src="../../Platform.Win32/Realm.Win32/bin/$Configuration$/Realm.*" target="lib/net452" />
+    <file src="../../wrappers/build/$Configuration$-Win32/realm-wrappers.dll" target="native/win32/x86" />
+    <file src="../../wrappers/build/$Configuration$-x64/realm-wrappers.dll" target="native/win32/x64" />
```

Comment:
We'll have to have a total overhaul of the nuget package for UWP where we'll use the new `runtimes` special folder and that handles UWP binaries with a special `uap` convention.